apps/frontend/package.json

```diff
@@ -21,6 +21,7 @@
     "bootstrap": "^5.3.0",
     "react": "^18.2.0",
     "react-bootstrap": "^2.9.0",
+    "react-date-picker": "^10.5.2",
```

Comment:
Is this used somewhere?

Reply:
No, initially I experimented with it for the dob field, but I ended up implementing my own date field. Will delete.